src/TesApi.Web/AzureBillingUtils.cs

```diff
@@ -0,0 +1,539 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TesApi.Web
+{
+    /// <summary>
+    /// Utility class to work with Azure Region names
+    /// </summary>
+    public static class AzureBillingUtils
+    {
+        /// <summary>
+        /// Lists all VM sizes verified supported by Azure Batch, along with the family and billing meter names
+        /// </summary>
+        /// <returns>List of VM sizes</returns>
+        public static List<(string VmSize, string FamilyName, string MeterName, string MeterSubCategory)> GetVmSizesSupportedByBatch()
+        {
+            return VmSizesFamiliesAndMeters.Where(v => VerifiedVmSizes.Contains(v.VmSize, StringComparer.OrdinalIgnoreCase)).ToList();
+        }
+
+        private static IEnumerable<(string VmSize, string FamilyName, string MeterName, string MeterSubCategory)> VmSizesFamiliesAndMeters => new[] {
```

Comment:
Did you generate this list with code?  If so, we should consider adding it in the repo somewhere, and also update our docs with instructions for how to generate this list

Reply:
Will add this when I clean up the code.